RoutingTests/OSRoutingServiceTests.swift

```diff
@@ -57,8 +58,9 @@ class OSRoutingServiceTests: XCTestCase {
             self.handler = completion
         }
 
-        func routeBetween(locations locations: [CLLocationCoordinate2D], completion: (Result<Route> -> Void)) {
-
+        func routeBetween(coordinates coordinates: [CLLocationCoordinate2D], completion: (Result<Route> -> Void)) {
```

Comment:
Asked Stuart about that, and the `/route` endpoint sanitises the input points for you. But agree that's certainly not clear in this implementation at the moment, so maybe that should be highlighted somehow. I will implement the `/nearest` api as well though for those that want to pre-sanitise points.